src/chargepoint/ConnectorData.h

```diff
@@ -82,6 +84,20 @@ struct ConnectorData
     bool unavailable_pending;
     /** @brief Meter */
     MeterSimulator* meter;
+
+
+    static inline std::string typeToString(Type p_type)
```

Comment:
No need to redefine this kind of functions.
Use the EnumFromStringToString  class.